src/splash/metag/index.md

```diff
@@ -2,6 +2,6 @@
 title: Latest Tutorials
 date: 2025-03-04
 ---
-Check out [the latest metagenomics tutorial](https://galaxyproject.github.io/training-material//Metagenomics/tutorials/mothur-miseq-sop) by [@bebatut](https://twitter.com/bebatut) and [@shiltemann](https://twitter.com/shiltemann)!
+Check out [the latest metagenomics tutorial](http://galaxyproject.github.io/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html) by [@bebatut](https://twitter.com/bebatut) and [@shiltemann](https://twitter.com/shiltemann)!
```

Comment:
This link can actually be now replaced with: `training.galaxyproject.org/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html`